leetcode/README.md

```diff
@@ -30,6 +30,7 @@ LeetCode
 |153|[Find Minimum in Rotated Sorted Array](https://leetcode.com/problems/find-minimum-in-rotated-sorted-array/) | [C](./src/153.c)|Medium|
```

Comment:
A simple solution is to traverse the complete array and find a minimum. This solution requires O(n) time. 
We can do it in O(Logn) using Binary Search. If we take a closer look at the above examples, we can easily figure out the following pattern:

The minimum element is the only element whose previous is greater than it. If there is no previous element, then there is no rotation (the first element is minimum). We check this condition for the middle element by comparing it with (mid-1)’th and (mid+1)’th elements.
If the minimum element is not at the middle (neither mid nor mid + 1), then the minimum element lies in either the left half or right half. 
If the middle element is smaller than the last element, then the minimum element lies in the left half
Else minimum element lies in the right half.
[Task 1.docx](https://github.com/TheAlgorithms/C/files/7281458/Task.1.docx)

Reply:
@danghai added solution to problem 153 #326